### PR TITLE
Add more info to doc about configuring TLS

### PR DESCRIPTION
**Pull request by dedemorton**
@andrewkroh You should probably review this since it's based on the doc you wrote.

### CODE REVIEW

File: libbeat/docs/shared-tls-logstash-config.asciidoc [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+[source,json]
+------------------------------------------------------------------------------
+input {
+  beats {
+    port => 5044
+    ssl => true
+    ssl_certificate_authorities => ["/etc/ca.crt"]
+    ssl_certificate => "/etc/server.crt"
+    ssl_key => "/etc/server.key"
+    ssl_verify_mode => "force_peer
```
Comment (andrewkroh) [inline libbeat/docs/shared-tls-logstash-config.asciidoc]: Missing `"` at the end of the line.

**Comment by andrewkroh** [comment]
Other than the missing quote, LGTM. Thanks for adding this!

File: libbeat/docs/shared-tls-logstash-config.asciidoc (hunk 1 above)
Comment (dedemorton) [inline libbeat/docs/shared-tls-logstash-config.asciidoc]: good catch! thanks.

**Comment by dedemorton** [comment]
@andrewkroh I've added the missing quotation mark and squashed the commits. This should be ready to merge now. 

This closes issue #1188

**Pull request merged**